apps/Launcher/qml/OptionsPanel.qml

```diff
@@ -54,7 +54,7 @@ DefaultPanel {
         }
         CheckBox {
             id: alphaBlending
-            text: "Alpha blending"
+            text: "Transparent Contents"
```

Comment:
Could work too, but note that background is primarily used to refer to the "Desktop" background which I fear makes it a bit confusing. It is strange that this simple feature is so hard to name clearly. Alternatively, what do you think about "Transparent Windows"?